.github/workflows/node.js.yml

```diff
@@ -1,34 +0,0 @@
-# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
-# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
-
-name: Build Frontend
-
-on:
-  push:
-    branches: [ "main" ]
-  pull_request:
-    branches: [ "main" ]
-    
-
-jobs:
-  build:
-    runs-on: ubuntu-latest
-    defaults:
-      run:
-        working-directory: frontend
-    strategy:
-      matrix:
-        node-version: [14.x, 16.x, 18.x, 21.x]
-        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
-
-    steps:
-    - uses: actions/checkout@v3
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v3
-      with:
-        node-version: ${{ matrix.node-version }}
-        cache: 'npm'
-        cache-dependency-path: '**/package-lock.json'
-    - run: npm ci
-    - run: NODE_OPTIONS=--max_old_space_size=8192 npm run build --if-present
-    - run: npm run test --if-present
```

Comment:
@sannidhishukla the PR description says to check this file but it is deleted -- it should be `.github/workflows/ci.yml` instead, right?